src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  box-sizing: border-box;
+}
+
+.search-bar {
+  position: relative;
+}
+
+.search-bar-big {
+  top: 12px;
+}
+
+.search-bar-small {
+  top: 102px;
+}
+
+.search-button,
+.search-field {
+  position: absolute;
+  outline: none;
+}
```

Comment:
let's give position: absolute only to button, leave search-field in the document flow